### PR TITLE
Add Accessibility Statement

### LINKED ISSUE #36: Add an Accessibility Statement Page
Create a page to outline the organization's commitment to accessibility and provide ways to contact the organization with feedback if users encounter an issue with using the site.

### PR DESCRIPTION
**Pull request by alexisreytech**
Still need Google Docs version.

Fixes #36

### CODE REVIEW

File: astro/src/pages/about/policies.astro [hunk 1]
```diff
@@ -28,6 +28,13 @@ const crumbs = [{
     develops. Our users should know that these policies exist and where to find them.
   </p>
   <ul>
+    <li>
+      <a href="../../accessibility-statement" class="text-primary">Accessibility Statement</a>
+      <!-- TODO: Add the link for Google Docs -->
+      <ExternalLink href="#" gaps={true}>
```
Comment (brian-montgomery) [inline astro/src/pages/about/policies.astro:34]: We're not going to have a google doc for this one. Please delete.

File: astro/src/pages/accessibility-statement.astro [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ <ThemedSection style="secondary">
+  <h2>Accessibility And Accommodations</h2>
+  <p>At Accessible Community, we believe that our commitment to creating accessible communities starts with us so we strive to provide accessible content and services.</p>
+  <p>We test our websites and services against
+   <ExternalLink
+    href="https://www.w3.org/TR/WCAG22/"
+    target="blank"
+    >WCAG 2.2 A, AA, and AAA criteria</ExternalLink>
+   and
+   <ExternalLink
```
Comment (brian-montgomery) [inline astro/src/pages/accessibility-statement.astro:17]: For links with a lot of text, please add the `inline={true}` attribute to the `ExternalLink` component, as it allows the text to wrap.

**Review by brian-montgomery — CHANGES_REQUESTED** [review]
Looks good. A couple of small changes.

File: astro/src/pages/accessibility-statement.astro (hunk 2 above)
Comment (alexisreytech) [reply astro/src/pages/accessibility-statement.astro:17]: I was wondering about that. Looks much better!

**Comment by alexisreytech** [comment]
@brian-montgomery, I made the requested changes. The only question is: Since it says all policies have links on Google Docs, should we change the wording for that?

**Review by brian-montgomery — APPROVED** [review]

**Pull request merged**